### PR TITLE
Add `worstAmount` to `EdgeSwapQuote`

### PR DESCRIPTION
**Pull request by Jon-edge**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Description

 none

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208162689994840

### CODE REVIEW

File: src/types/types.ts [hunk 1]
```diff
@@ -791,7 +791,6 @@ export interface EdgeGetTransactionsOptions {
   startDate?: Date
   endDate?: Date
   searchString?: string
-  spamThreshold?: string
```
Comment (paullinator) [inline src/types/types.ts:794]: why was this removed?

File: src/types/types.ts [hunk 2]
```diff
@@ -1348,6 +1344,11 @@ export interface EdgeSwapQuote {
    * Maximum amount of time this quote will take to be fulfilled (in seconds)
    */
   readonly maxFulfillmentSeconds?: number
+
+  /** Worse case amount - the highest cost for a 'from' quote, and the lowest
+   * receive amount for a 'to' quote */
+  readonly worstAmount?: string
```
Comment (paullinator) [inline src/types/types.ts:1350]: Change to minAmount. Regardless of the quote being `from` or `to`, this will always be the minimum RECEIVE amount. I'd be ok renaming to `minReceiveAmount`

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/types/types.ts (hunk 1 above)
Comment (Jon-edge) [reply src/types/types.ts:794]: Rebase issue, will re-instate

**Review by paullinator — APPROVED** [review]

File: src/types/types.ts [hunk 3]
```diff
@@ -818,9 +817,6 @@ export interface EdgeStreamTransactionOptions {
   /** Only return transactions matching this string */
   searchString?: string
 
-  /** Filter incoming transactions with a `nativeAmount` below this */
-  spamThreshold?: string
-
```
Comment (swansontec) [inline src/types/types.ts:823]: This is still missing. Please put it back when you rebase.

**Review by swansontec — CHANGES_REQUESTED** [review]

**Review by swansontec — APPROVED** [review]
